addons/garage/functions/fnc_onTextureSelect.sqf

```diff
@@ -26,5 +26,12 @@ for "_i" from 0 to (lbSize _ctrlListTextures - 1) do {
 // Check selected texture
 _ctrlListTextures lbSetPicture [_selectedIndex, ICON_CHECKED];
 
+private _texture = _ctrlListTextures lbData _selectedIndex;
+
+// Handle serialized array
+if (_texture != "" && {_texture select [0, 1] == '['}) then {
+    _texture = parseSimpleArray _texture;
```

Comment:
Use `getVariable` combined with `lbData` instead.

Reply:
Not sure what you mean. I could replace `lbData` with `getVariable`,.

Reply:
Nvm, I took inspiration from your approach for the reinforcement module